projects/schematics/src/shared/lib-configs/integration-libs/opf-schematics-config.ts

```diff
@@ -45,44 +39,9 @@ export const OPF_BASE_TRANSLATIONS = 'opfBaseTranslations';
 export const OPF_BASE_TRANSLATION_CHUNKS_CONFIG =
   'opfBaseTranslationChunksConfig';
 
-export const OPF_CHECKOUT_SCHEMATICS_CONFIG: SchematicConfig = {
-  library: {
-    featureName: OPF_CHECKOUT_FEATURE_NAME,
-    mainScope: SPARTACUS_OPF,
-    featureScope: SPARTACUS_OPF_CHECKOUT,
-  },
-  folderName: OPF_FOLDER_NAME,
-  moduleName: OPF_MODULE_NAME,
-  featureModule: {
-    name: OPF_CHECKOUT_MODULE,
-    importPath: SPARTACUS_OPF_CHECKOUT,
-  },
-  rootModule: {
-    name: OPF_CHECKOUT_ROOT_MODULE,
-    importPath: SPARTACUS_OPF_CHECKOUT_ROOT,
-  },
-  i18n: {
-    resources: OPF_CHECKOUT_TRANSLATIONS,
-    chunks: OPF_CHECKOUT_TRANSLATION_CHUNKS_CONFIG,
-    importPath: SPARTACUS_OPF_CHECKOUT_ASSETS,
-  },
-  styles: {
-    scssFileName: OPF_SCSS_FILE_NAME,
-    importStyle: SPARTACUS_OPF,
-  },
-  dependencyFeatures: [CHECKOUT_BASE_FEATURE_NAME],
-  importAfter: [
-    {
-      markerModuleName: CHECKOUT_BASE_MODULE,
-      featureModuleName: OPF_CHECKOUT_MODULE,
-    },
-  ],
-  customConfig: buildOpfCheckoutConfig,
-};
-
 export const OPF_BASE_SCHEMATICS_CONFIG: SchematicConfig = {
```

Comment:
I wonder if should this be named `OPF_SCHEMATICS_CONFIG `, and the function `buildOpfBaseConfig` -> `buildOpfConfig`?